Source/VMware.vSphereDSC/DSCResources/VMHostNetwork/VMHostNetworkMigration/VMHostVDSwitchMigration.ps1

```diff
@@ -371,6 +384,45 @@ class VMHostVDSwitchMigration : VMHostNetworkMigrationBaseDSC {
     <#
     .DESCRIPTION
 
+    Creates a new Distributed Port Group with the specified name on the specified Distributed Switch.
+    If a Standard Port Group with the same name exists on the Standard Switch from which the VMKernel Network Adapters are migrated
+    and the Standard Port Group is associated with a VLAN, the new Distributed Port Group is created with the same VLAN to avoid
+    any network misconfigurations due to VLANs not configured correctly.
+    #>
+    [PSObject] CreateDistributedPortGroup($distributedPortGroupName, $distributedSwitch, $vmKernelNetworkAdapters) {
+        try {
+            $createVDPortGroupMessageEnd = "."
+            $newVDPortGroupParams = @{
+                Server = $this.Connection
+                Name = $distributedPortGroupName
+                VDSwitch = $distributedSwitch
+                Confirm = $false
+                ErrorAction = 'Stop'
+                Verbose = $false
+            }
+
+            $vmKernelNetworkAdapterOfStandardPortGroup = $vmKernelNetworkAdapters | Where-Object -FilterScript { $_.PortGroupName -eq $distributedPortGroupName }
+            if ($null -ne $vmKernelNetworkAdapterOfStandardPortGroup) {
+                $standardPortGroup = $this.GetStandardPortGroup($distributedPortGroupName)
+
+                # The VLAN for Distributed Port Groups is valid only in the specified range.
+                if ($standardPortGroup.VLanId -In 1..4094) {
+                    $newVDPortGroupParams.VlanId = $standardPortGroup.VLanId
+                    $createVDPortGroupMessageEnd = " with VLAN ID $($standardPortGroup.VLanId)."
+                }
+            }
+
+            Write-VerboseLog -Message $this.CreateVDPortGroupMessage -Arguments @($distributedPortGroupName, $distributedSwitch.Name, $createVDPortGroupMessageEnd)
```

Comment:
Here as well the **Write-Verbose** will need to be replaced with **WriteLogUtil** method call when the streams PR gets merged.